### PR TITLE
20180917 domain updated field

### PR DESCRIPTION
**Pull request by peterthomassen**

### CODE REVIEW

File: docs/domains.rst [hunk 1]
```diff
@@ -62,6 +70,18 @@ Field details:
     characters as well as hyphens ``-`` and underscores ``_`` (except at the
     beginning of the name).  The maximum length is 191.
 
+``published``
+    :Access mode: read-only
+
+    Timestamp of when the domain's DNS records have lasted been published,
```
Comment (nils-wisiol) [inline docs/domains.rst]: typo in "lasted"

File: docs/domains.rst [hunk 2]
```diff
@@ -62,6 +70,18 @@ Field details:
...
+    in ISO 8601 format (e.g. ``2013-01-29T12:34:56.000000Z``).
+
+    As we publish record modifications immediately, this usually indicates
```
Comment (nils-wisiol) [inline docs/domains.rst]: remove "usually" (the below-mentioned exception is IMHO not important enough to mention it twice)

File: test/e2e/schemas.js [hunk 3]
```diff
@@ -26,8 +26,10 @@ exports.domain = {
             minItems: 1
         },
         name: { type: "string" },
+        created: { type: "string" },
+        published: { type: "string" },
```
Comment (nils-wisiol) [inline test/e2e/schemas.js]: require datetime instead of string? (twice)

File: api/desecapi/serializers.py [hunk 4]
```diff
@@ -227,7 +227,10 @@ class UserSerializer(djoserSerializers.UserSerializer):
     class Meta(djoserSerializers.UserSerializer.Meta):
         fields = tuple(User.REQUIRED_FIELDS) + (
             User.USERNAME_FIELD,
+            'dyn',
+            'limit_domains',
         )
+        read_only_fields = ('dyn', 'limit_domains',)
```
Comment (nils-wisiol) [inline api/desecapi/serializers.py]: These fields are missing documentation; also I'm considering to remove/alter the dyn flag before we make the API stable. It's seems a odd category for users and I think should be replaced with something more general. The flag is currently used to determine which domain names are acceptable for any given user to register and in order to determine which emails to send out.

File: docs/authentication.rst [hunk 5]
```diff
@@ -134,8 +134,55 @@ header::
 The server will delete the token and respond with ``204 No Content``.
 
 
-Additional Tokens
-~~~~~~~~~~~~~~~~~
+Manage Account
+~~~~~~~~~~~~~~
+
+Change Email Address
```
[the remark this answers is not in the document]
Comment (peterthomassen) [reply docs/authentication.rst]: done

File: docs/domains.rst (hunk 1 above)
Comment (peterthomassen) [reply docs/domains.rst]: done

File: test/e2e/schemas.js (hunk 3 above)
Comment (peterthomassen) [reply test/e2e/schemas.js]: done

File: api/desecapi/serializers.py (hunk 4 above)
Comment (peterthomassen) [reply api/desecapi/serializers.py]: done

**Comment by peterthomassen** [comment]
- Docs have been changed, section about user object JSON format (`limit_domains` etc.) has been added to the previous commit (sorry, didn't think before squashing). Please take a look still.
- JSON formation validation has been added
- simplified docs compilation

pls review again - thx

**Pull request merged**